app/templates/briefs/view_response_result.html

```diff
@@ -148,8 +148,8 @@ <h2 class="summary-item-heading">What happens next</h2>
 
   {%
     with
-      url = url_for(".dashboard"),
-      text = "Your account"
+      url = url_for(".opportunities_overview", framework_slug=brief.frameworkSlug),
```

Comment:
But we should never make the work "dashboard" visible on the site or in any URLs.